src/test/java/seedu/address/model/property/FilterPropsByOwnerNamePredicteTest.java

```diff
@@ -40,36 +39,40 @@ public void equals() {
 
     @Test
     public void test_propertyOwnerNameMatch_returnsTrue() {
-        Owner rickyTan = new Owner(new Name("Ricky Tan"), new Phone("1324019384"));
-        Owner kim = new Owner(new Name("Kim"), new Phone("124801324"));
+        String rickyName = "Ricky Tan";
+        String rickyPhone = "1324019384";
+        String kimName = "Kim";
+        String kimPhone = "124801324";
```

Comment:
Could probably take this out of the method, since both methods in this class are using the same variables